app/index.js

```diff
@@ -13,7 +13,7 @@ ComposerGenerator = module.exports = function ComposerGenerator(args, options) {
     this.installDependencies({ skipInstall: options['skip-install'] });
   });
 
-  this.pkg = JSON.parse(this.readFileAsString(path.join(__dirname, '../package.json')));
+  this.pkg = require("../package.json");
```

Comment:
yeah, why the hell could this line 16 have ever existed in such a horrible way before this change!